.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python-version: [2.7, 3.5, 3.6, 3.7, 3.8]
```

Comment:
We cannot drop Python 2 support that easily, see #370.

Making the tests Python 2 compatible should be possible from what I've seen. It's only the syntax for `super()` calls and the `to_bytes` conversion.

Reply:
For `super()` I copied the syntax from blocks.py which didn't break on py2.7 and now it works.
I don't fully understand the `super()` problem, but it's gone now.